Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -22,7 +23,7 @@
 namespace Emby.Server.Implementations.Updates
 {
     /// <summary>
-    /// Manages all install, uninstall and update operations (both plugins and system)
+    /// Manages all install, uninstall and update operations (both plugins and system).
```

Comment:
```suggestion
    /// Manages all install, uninstall, and update operations (both plugins and system).
```

Reply:
TBF, a comma isn't needed before "and".

Reply:
That depends on your stance about the Oxford comma, I'm a firm believer.